### PR TITLE
Audio small patch

### PR DESCRIPTION
**Pull request by yunlong-wang-cn**
1. add one more threshold to filter out the no-speech noise or the background speech noise by the  `no_speech_preb` reported by Whisper.
2. remove the test log
3. add reference value for onset detection, to enhance the detection of music note from right zone of the marimba, also it has the function of the denoise.

### CODE REVIEW

File: marimbabot_bringup/launch/marimbabot.launch [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <launch>
   <include file="$(find marimbabot_vision)/launch/marimbabot.launch"/>
-  <include file="$(find marimbabot_audio)/launch/marimbabot.launch"/>
-  <include file="$(find marimbabot_planning)/launch/marimbabot.launch"/>
   <include file="$(find marimbabot_speech)/launch/marimbabot.launch"/>
   <include file="$(find marimbabot_behavior)/launch/marimbabot.launch"/>
+  <include file="$(find marimbabot_audio)/launch/marimbabot.launch"/>
```
Comment (Juphex) [inline marimbabot_bringup/launch/marimbabot.launch:5]: Why did you change the order?

Comment (yunlong-wang-cn) [reply marimbabot_bringup/launch/marimbabot.launch:5]: they though it's much intuitive.  I forgot who made this change，since they decide to do this,  I just put it here. But from my perspective its doesnt matter.

File: marimbabot_audio/src/onset_detection.py [hunk 2]
```diff
@@ -127,7 +127,8 @@ def init_detection_config(self):
 			confidence threshold for note classification(crepe)
 		"""
 		# For onset detection
-		self.confidence_threshold = 0.7  # the threshold for note classification
+		self.confidence_threshold = rospy.get_param("~confidence_threshold", 0.7)
```
Comment (Flova) [inline marimbabot_audio/src/onset_detection.py:130]: I would not include a default, as it might cause confusions when somebody changes that.

**Review by Flova — APPROVED** [review]

File: marimbabot_audio/src/onset_detection.py (hunk 2 above)
Comment (yunlong-wang-cn) [reply marimbabot_audio/src/onset_detection.py]: done
